.github/workflows/ci.yml

```diff
@@ -28,30 +28,21 @@ jobs:
     needs: check_if_skip_duplicate_job
     if: needs.check_if_skip_duplicate_job.outputs.should_skip != 'true'
     steps:
-      #- uses: cachix/install-nix-action@v22
       - uses: actions/checkout@v4
       - run: nix build -L .#checks.x86_64-linux.aeneas-check-tidiness
       - run: nix build -L .#aeneas
       - run: nix build -L .#checks.x86_64-linux.aeneas-tests
       - run: nix build -L .#checks.x86_64-linux.aeneas-verify-fstar
       - run: nix build -L .#checks.x86_64-linux.aeneas-verify-coq
       - run: nix build -L .#checks.x86_64-linux.aeneas-verify-hol4
-      # Lean doesn't work with Nix
+      # Lean cannot run its tests in the sandbox because `elan` will download things
       #- run: nix build -L .#checks.x86_64-linux.aeneas-verify-lean
 
-  lean: # Lean isn't supported by Nix, so we put it in a different job
+  lean:
     runs-on: [ubuntu-latest]
```

Comment:
why not run it on our self-hosted machine?

Reply:
you lost parallelism

Reply:
fair